docs/4.1/components/pagination.md

```diff
@@ -82,19 +80,19 @@ You can optionally swap out active or disabled anchors for `<span>`, or omit the
 {% capture example %}
 <nav aria-label="...">
   <ul class="pagination">
-    <li class="page-item disabled">
+    <li class="page-item disabled" aria-disabled="true">
```

Comment:
as there's nothing focusable nor operable inside here, `aria-disabled` is mostly useless here